init_x: 0  # initial x-pos 
init_y: 0  # initial y-pos
init_z: 0  # initial z-pos

vehicle_l: 3  # vehicle list for visualization
vehicle_num: $(arg vehicle_num)

# number of random targets & obstalces
# -1 if providing custom list of targets
n_rand_obst: -1

del_t: .02  # timestep between sim updates

waypt_threshold: 3.0

obstacles:
    - {id: 3, x: 15,  y: 15,  width: 2.0, length: 6.0, height: 20.0, speed: 0.0, hold_heading_time: 1.0}
    - {id: 2, x: -15,  y: 15,  width: 2.0, length: 6.0, height: 20.0, speed: 0.0, hold_heading_time: 1.0}
    - {id: 1, x: -15,  y: -15,  width: 1.0, length: 6.0, height: 20.0, speed: 0.0, hold_heading_time: 1.0}
